src/web/services/simple-modal.service.ts

```diff
@@ -66,4 +66,14 @@ export class SimpleModalService {
     };
     return this.open(header, type, content, modalOptions);
   }
+
+  openDownloadModal(header: string, type: SimpleModalType,
```

Comment:
Maybe rephrase this to `openLoadingModal`? Since this is a generic service that can be reused for other functions.